test/e2e/metrics_e2e_test.go

```diff
@@ -266,6 +270,109 @@ var _ = Describe("Metrics are generated for OLM managed resources", func() {
 			})
 		})
 	})
+
+	Context("Metrics emitted by CatalogSources", func() {
+		When("A valid CatalogSource object is created", func() {
+			var (
+				name        = "metrics-catsrc-valid"
+				cleanup     func()
+				cleanupDone = false
```

Comment:
You'll want to set this to `true` in a `BeforeEach` -- this assignment will only happen once at package initialization time.

Reply:
Right, that's what happens here https://github.com/operator-framework/operator-lifecycle-manager/pull/2152/files#diff-d5a3dea398f2933b707ed901c793832f40784c0999a861847356f823e536c5d4R324-R326

Reply:
@benluddy do you mean like it's being set here https://github.com/operator-framework/operator-lifecycle-manager/pull/2152/files#diff-d5a3dea398f2933b707ed901c793832f40784c0999a861847356f823e536c5d4R324-R327 or do you have something else in mind?